docs/antora/modules/ROOT/pages/management/docker.adoc

```diff
@@ -10,10 +10,23 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_config: configuration/docker.adoc
 :url_FLUSHALL: https://redis.io/commands/FLUSHALL
 
+NOTE: To upgrade Lisk Service to a desired version, please check out the specific version with *`git checkout <tag>`*, and then follow the existing process described below.
+
+== Build Docker image
+
+.Inside the lisk-service root folder
+----
+make build
+----
+This creates the necessary Docker images to start Lisk Service in the containers.
+
 == Start
 
 .Inside the lisk-service root folder
 ----
+cp ./docker/example.env .env
```

Comment:
i don't think this needs t be done every time Lisk Service is started, so it doesn't really fit here.

Reply:
but this is not about starting it for the first time, it is about how to start it in general. I don't think you need to do it every time you start Lisk Service. So it shouldn't be mentioned here

Reply:
As mentioned, this is not about starting for the first time. This step is already included at the right location, which is the setup guides: https://liskhq.github.io/lisk-docs/lisk-service/0.7/setup/docker.html#connecting-lisk-service-to-a-blockchain-node